test/edu/csus/ecs/pc2/core/report/ContestCompareModelTest.java

```diff
@@ -195,12 +195,16 @@ public void testReport() throws Exception {
             // only test if there is a pretest dir found
             return;
         }
+        // Load Accounts
+        String loadAccountFilename = "/repos/icpc/ccsconfig-dhaka/contests/pretest2/config/load_accounts_real.tsv";
```

Comment:
I do not understand why there is an assumption made that a file from Dhaka exists on the machine where the tests are being run.  Even if such a file/path does exist on the author's machine, this test is going to fail if run on somebody else's machine unless the specified file is committed to the master repository (and I don't see any such thing anywhere...)
Please either remove this, or explain why it is important and how it is supposed to work.